src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -1348,7 +1348,7 @@ Convert::PitchStruct Convert::pitchFromMEI(const libmei::Note& meiNote, const li
 
     int step = meiNote.HasPname() ? meiNote.GetPname() - 1 : 0;
     // It should never be necessay, but just in case
-    std::clamp(step, 0, 6);
+    (void)std::clamp(step, 0, 6);
```

Comment:
I think this is rather intended to be:
```suggestion
    step = std::clamp(step, 0, 6);
```

Reply:
thanks, fixed